package.json

```diff
@@ -1,29 +1,31 @@
 {
   "name": "hyperverse",
-  "version": "1.0.0",
+  "version": "0.0.0",
```

Comment:
Can we keep it at 1 please?

Reply:
Yes, it'll be in the first *proper* release, with the `tag` and `changeLog`